### DISCUSSION TITLE
Route other subnets?

### DISCUSSION THREAD

**Discussion by deanfourie1** [homeassistant-apps/app-cloudflared#740] [Q&A]
Hi there,

I have another tunnel, on another subnet. Is it possible to route an entire subnet or single host though to another tunnel. This subnet is setup as a tunnel with a hostname too, and has a local subnet added to it.

ie.

192.168.1.0/24 or 192.168.1.1

Thanks

**Comment by brenner-tobias** [comment]
Hi. Since this is specific to Cloudflared and not this add-on, I suggest you have a look at the [official documentation](https://developers.cloudflare.com/cloudflare-one/connections/connect-networks/) and their [community](https://community.cloudflare.com/), which is very active.
